### PR TITLE
feat(suite-native): sort networks according to networksConfig

### LINKED ISSUE #15222: Order networks by networkConfig
Currently mobile app uses custom ordering of networks in coin enabling, dashboard and portfolio tracker . Remove this and use ordering based on how networks are ordered in the [networksConfig](https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-config/src/networksConfig.ts).

### PR DESCRIPTION
**Pull request by vytick**
Use networksConfig order of networks to sort networks in the app.

## Related Issue

Resolve #15222

### CODE REVIEW

File: suite-native/config/src/supportedNetworks.ts [hunk 1]
```diff
@@ -48,10 +49,12 @@ export const discoverySupportedNetworks = [
     ...networkSymbolsWhitelistMap.testnet,
 ];
 
-export const sortNetworks = (networks: Network[]) =>
-    A.sort(networks, (a, b) => {
-        const aOrder = discoverySupportedNetworks.indexOf(a.symbol) ?? Number.MAX_SAFE_INTEGER;
-        const bOrder = discoverySupportedNetworks.indexOf(b.symbol) ?? Number.MAX_SAFE_INTEGER;
+export const orderedNetworksSymbols = Object.keys(networks) as NetworkSymbol[];
```
Comment (yanascz) [inline suite-native/config/src/supportedNetworks.ts]: I'd go with this name:
```suggestion
export const orderedNetworkSymbols = Object.keys(networks) as NetworkSymbol[];
```

Comment (vytick) [reply suite-native/config/src/supportedNetworks.ts]: 546048a009

**Review by yanascz — APPROVED** [review]

**Pull request merged**